man/systemd.exec.xml

```diff
@@ -2937,6 +2937,20 @@ StandardInputData=SWNrIHNpdHplIGRhIHVuJyBlc3NlIEtsb3BzLAp1ZmYgZWVtYWwga2xvcHAncy
         <varname>TTYPath=</varname> before and after execution. Defaults to <literal>no</literal>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>TTYRows=</varname></term>
+
+        <listitem><para>Configure the number of rows for the TTY specified with <varname>TTYPath=</varname>.
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>TTYColumns=</varname></term>
```

Comment:
maybe merge these two entries into one, i.e.

```xml
     <varlistentry>
        <term><varname>TTYRows=</varname></term>
        <term><varname>TTYColumns=</varname></term>

        <listitem><para>Configure the number of rows/columns for the TTY specified with <varname>TTYPath=</varname>.
        </para></listitem>
      </varlistentry>
```

They are not independent options, but closely related settings